### DISCUSSION TITLE
Is there any way to make Search box always visible

### DISCUSSION THREAD

**Discussion by esledov** [keepassxreboot/keepassxc#9784] [Q&A] [answered]
I am using KeepassXC 2.7.6 on Ubuntu. 

Every time I open the app, I need to press Ctrl-F to open the search box. And it looks like a layer on top of the main interface. Is there an option to make it always visible?

Wouldn't it make sense to keep the search box always visible? When I open a browser, I don't need to use a shortcut to see the address line.

**Comment by droidmonkey** [comment, marked as answer]
Go to application settings, scroll to bottom, change the toolbar to be icons only

**Comment by JeckPeppinger** [comment]
the toolbar indeed is not configurable and extremely chunky. all the commands and search field i need are always hidden except for fully maxing the window. suggest this will be designed more flexible and not so chunky.
